public/unsupported-browser/browser-check.js

```diff
@@ -0,0 +1,12 @@
+function browserCheck() {
+  // https://github.com/w3c/csswg-drafts/issues/3559#issuecomment-1758459996
+  const supportsFlexGap = typeof CSS !== "undefined" ? CSS.supports("inset: 0") : false;
```

Comment:
Since this can run on old browsers, to be safe, maybe better to use `var` instead of `const`?

Reply:
haha yeah i guess we should keep IE10 in mind :) I changed it :+1: